### PR TITLE
Copy over all config files

### PR DESCRIPTION
**Pull request by hobinjk**

### CODE REVIEW

**Comment by hobinjk** [comment]
Notably this doesn't restart things to make sure that changes in config files are propagated. At least for dnsmasq the gateway handles starting/stopping so that isn't a concern.

File: tools/pre-upgrade.sh [hunk 1]
```diff
@@ -28,3 +28,5 @@ sudo pip3 install -U "$_url"
 # Upgrade adapt-parser Python package
 _url="git+https://github.com/mycroftai/adapt#egg=adapt-parser"
 sudo pip3 install -U "$_url"
+
+sudo cp -r image/etc/ /
```
Comment (mrstegeman) [inline tools/pre-upgrade.sh]: We should probably do some proper permissions.

```bash
sudo chown -R root:root "image/etc"
sudo cp -r "image/etc" /
sudo chown -R $(id -u):$(id -g) "image/etc"
```

Comment (hobinjk) [reply tools/pre-upgrade.sh]: :+1:, feel free to continue your PR and obsolete this one :)

Comment (mrstegeman) [reply tools/pre-upgrade.sh]: Nah, already deleted it. These were my changes exactly.

Comment (hobinjk) [reply tools/pre-upgrade.sh]: Okay, updated this PR with those changes

**Pull request merged**